### ISSUE TITLE
Can you share the code for training imagenet?

### ISSUE THREAD

**Issue by YehLi** [d-li14/PSConv#8]
Thanks for your great work. Can you share the code for training imagenet? I use the code (https://github.com/rwightman/pytorch-image-models) and train ResNeXt-50+PSConv with the same setting in your paper (SGD, 120 epochs, half cosine function shaped schedule, cross entropy, standard augmentation, weight decay 1e-4). The top1 performance is only 77.57%, which is lower than the results reported in your paper (79.622%).
